tests/meta/backup/test_meta_backup.py

```diff
@@ -32,9 +32,7 @@ def test_maximum_processing_queue_full(mock_store):
     )
 
     # WHEN there's already a flow cell being retrieved from PDC
-    mock_store.get_flow_cells_by_statuses(
-        flow_cell_statuses=[FlowCellStatus.PROCESSING]
-    ).count.return_value = 1
+    mock_store.get_flow_cells_by_statuses.return_value = [[mock.Mock()]]
```

Comment:
What does this solve?

Reply:
Initially there was a query passed. Now it is a list of flow cell. 
When using just mock.Mock() it only returns a mock object, but for the logic to work it needs to be a list of mock. So that there will be a len() greater than 0 if something is passed which this test checks for.